packages/genesys-spark-components/src/utils/date/time-zone.spec.ts

```diff
@@ -17,84 +17,10 @@ describe('#formattOffset', () => {
 });
 
 describe('#getTimeZoneList', () => {
-  // Spot check zones rather than entire list, because this will change depending on the version of node being run, and thus would require an exact node version for each local test run
   it('should should retrieve the list', () => {
     jest.setSystemTime(new Date('2023-09-26'));
     const output = getTimeZoneList();
 
     expect(output.length).toEqual(576);
```

Comment:
I think this `expect` needs to be removed aswell

Reply:
No that is fine. It is consistent for a specific version of noew.